system-int-test/test/cmr/system_int_test/search/variable/variable_association_test.clj

```diff
@@ -111,49 +112,45 @@
   (e/grant-registered-users (s/context) (e/coll-catalog-item-id "PROV1"))
   (let [native-id "var123"
         token (e/login (s/context) "user1")
-        {:keys [concept-id revision-id]} (vu/ingest-variable-with-attrs {:native-id native-id
-                                                                         :Name "variable1"})
+        var-concept (vu/make-variable-concept {:native-id native-id
+                                               :Name "variable1"
+                                               :provider-id "PROV1"})
+        {:keys [concept-id revision-id]} (vu/ingest-variable var-concept)
         coll-concept-id (:concept-id (d/ingest "PROV1" (dc/collection)))]
     (testing "Associate variable using query sent with invalid content type"
       (are [associate-variable-fn request-json]
         (= {:status 400,
             :errors
             ["The mime types specified in the content-type header [application/xml] are not supported."]}
            (associate-variable-fn token concept-id request-json {:http-options {:content-type :xml}}))
-        ; vu/associate-by-query {:provider "foo"}
-        vu/associate-by-concept-ids [{:concept-id coll-concept-id}]))
+
+        au/associate-by-concept-ids [{:concept-id coll-concept-id}]))
 
     (testing "Associate applies JSON Query validations"
       (are [associate-variable-fn request-json message]
         (= {:status 400
             :errors [message]}
            (associate-variable-fn token concept-id {:foo "bar"}))
 
-        ; vu/associate-by-query {:foo "bar"}
-        ; "/condition object instance has properties which are not allowed by the schema: [\"foo\"]"
-
-        vu/associate-by-concept-ids {:concept-id coll-concept-id}
+        au/associate-by-concept-ids {:concept-id coll-concept-id}
         "instance type (object) does not match any allowed primitive type (allowed: [\"array\"])"))
 
     (testing "Associate variable that doesn't exist"
       (are [associate-variable-fn request-json]
         (= {:status 404
             :errors ["Variable could not be found with concept id [V12345-PROV1]"]}
            (associate-variable-fn token "V12345-PROV1" request-json))
-        ; vu/associate-by-query {:provider "foo"}
-        vu/associate-by-concept-ids [{:concept-id coll-concept-id}]))
-
-    ;; This test is commented out since we don't support delete variable yet
-    ;; Once we support delete variable, re-enable this test.
-    ;; See CMR-4168
```

Comment:
Can we mark CMR-4168 as fixed?

Reply:
It is already marked as "Verified Internal".